### ISSUE TITLE
Windows: Didn't come with react-css-modules installed

### ISSUE THREAD

**Issue by glcheetham** [stylesuxx/generator-react-webpack-redux#52]
I selected "Y" for "Do you want to use CSS modules?" during setup and `react-css-modules` was not installed for me, causing this build error when I tried to build a component containing `import cssmodules from 'react-css-modules';`

```
ERROR in ./src/components/TextInput.js
Module not found: Error: Can't resolve 'react-css-modules' in 'C:\Users\glcheetham\Documents\Visual Studio 2015\Projects
\MyProject\MyProject\Content\MyReduxApplication\src\components'
 @ ./src/components/TextInput.js 15:23-51
 @ ./src/components/App.js
 @ ./src/containers/App.js
 @ ./src/index.js
 @ multi main
```

---
Environment Information:
```
PS C:\Users\glcheetham> (Get-WmiObject -class Win32_OperatingSystem).Caption
Microsoft Windows 10 Home
PS C:\Users\glcheetham> node --version
v6.9.1
PS C:\Users\glcheetham> npm --version
3.8.0
PS C:\Users\glcheetham> $PSVersionTable.PSVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      14393  206



PS C:\Users\glcheetham>
```

**Comment by stylesuxx** [comment]
Can you see it in the *node-modules* folder? Could you post any errors
encountered during **npm install**.

Am 10.11.2016 12:19 schrieb "glcheetham" <notifications@github.com>:

**Comment by glcheetham** [comment]
It wasn't in the node_modules folder or `package.json` out of the box.

No untoward errors from `npm install` other than that good ol' npm Windows cliché
```
PS > npm install
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.15
npm WARN webpack-dev-server@2.0.0-beta requires a peer of webpack@>=2.0.3-beta <3 but none was installed.
```

**Comment by stylesuxx** [comment]
OK. Thank you, I will look into it, might be a problem with the base
generator.

Am 10.11.2016 12:26 schrieb "glcheetham" <notifications@github.com>:

**Comment by stylesuxx** [comment]
I had time to look into this, it is a problem with the base generator, it has been fixed in master, but not the latest tagged version.

**Comment by bairisuresh** [comment]
Hi @stylesuxx 
Please let me know the version number for this fix . I got the same error.

**Comment by stylesuxx** [comment]
Fixed in 1.1.0 - feel free to re-open if problem persists.
